test/runtests.jl

```diff
@@ -7,296 +6,296 @@ using Einsum
 
 ## Test that vars in Main aren't overwritten by einsum
 let
-  i = -1
-  y = randn(10)
-  @einsum x[i] := y[i]
-  @test i == -1
+    i = -1
+    y = randn(10)
+    @einsum x[i] := y[i]
+    @test i == -1
 end
 
 ## Test that B is overwritten by := operator
 let
```

Comment:
If energetic it might be nice to replace all these let blocks with things like
```julia
@testset "Test that B is overwritten by := operator" begin
...
end
```

Reply:
That's definitely a good idea.